cli/popper/commands/cmd_ci.py

```diff
@@ -5,6 +5,11 @@
 from popper import scm, utils as pu
 from popper.cli import pass_context, log
 
+install_scripts = {
+    'singularity': """wget http://neurodeb.pirsquared.org/\
+pool/main/s/singularity-container/\
+singularity-container_2.6.1-2~nd16.04+1_amd64.deb && \
+sudo apt-get -f install ./singularity-container_2.6.1-2~nd16.04+1_amd64.deb"""}
```

Comment:
this is only going to work on Travis and probably CircleCI, as those are the only CI workers we know it has ubuntu/debian installed. In the other cases, we either don't know (Jenkins) or the OS is not debian-based (Gitlab). So what we can do is only enable this for Circle/Travis and show an error message on Jenkins/Gitlab.